<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>969</width>
    <height>698</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="Ekran" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>640</width>
      <height>640</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="reset">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>580</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Resetuj</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>20</y>
      <width>271</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string>Przesuwanie</string>
    </property>
    <widget class="QSlider" name="x_slider">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>160</width>
       <height>25</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-320</number>
     </property>
     <property name="maximum">
      <number>320</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QSlider" name="y_slider">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>160</width>
       <height>25</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-320</number>
     </property>
     <property name="maximum">
      <number>320</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>35</y>
       <width>49</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>X</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>85</y>
       <width>49</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Y</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>390</y>
      <width>270</width>
      <height>131</height>
     </rect>
    </property>
    <property name="title">
     <string>Pochylenie</string>
    </property>
    <widget class="QSlider" name="shx_slider">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>160</width>
       <height>25</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-100</number>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QSlider" name="shy_slider">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>160</width>
       <height>25</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-100</number>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>85</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Względem Y</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>35</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Względem X</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>240</y>
      <width>270</width>
      <height>141</height>
     </rect>
    </property>
    <property name="title">
     <string>Skalowanie</string>
    </property>
    <widget class="QSlider" name="sx_slider">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>160</width>
       <height>25</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>500</number>
     </property>
     <property name="value">
      <number>50</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QSlider" name="sy_slider">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>160</width>
       <height>25</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>500</number>
     </property>
     <property name="value">
      <number>100</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>37</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Względem X</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>87</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Względem Y</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>150</y>
      <width>271</width>
      <height>80</height>
     </rect>
    </property>
    <property name="title">
     <string>Obracanie</string>
    </property>
    <widget class="QSlider" name="alfa_slider">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>38</y>
       <width>160</width>
       <height>25</height>
      </rect>
     </property>
     <property name="maximum">
      <number>360</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="labelObracanie">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>30</y>
       <width>71</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>969</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Ekran</class>
   <extends>QWidget</extends>
   <header>ekran.h</header>
   <container>1</container>
   <slots>
    <signal>setCurrentAlpha(int)</signal>
    <slot>alphaSliderChange(int)</slot>
    <slot>warstwaButton()</slot>
    <slot>alphaSliderChange()</slot>
    <slot>setAlpha(int)</slot>
    <slot>blendMode()</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
